rp2040-hal/src/uart/utils.rs

```diff
@@ -84,6 +83,23 @@ pub struct UartConfig {
     pub parity: Option<Parity>,
 }
 
+impl UartConfig {
+    /// Create a new instance of UartConfig
+    pub fn new(
```

Comment:
Would it make sense to make this method const ?

Reply:
Good point, that may be useful.